scripts/ai2-internal/run_with_beaker.py

```diff
@@ -63,17 +61,40 @@ def main(param_file, description):
             '/output',
             "--source",
             f"{config_dataset_id}:/config.json",
-            f'--desc={description}',
-            '--gpu-count=1',
-            '--detach'] + [image] + allennlp_command
+            '--gpu-count=1'] + extra_beaker_commands + [image] + allennlp_command
     print(' '.join(command))
     subprocess.run(command, check=True)
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
 
     parser.add_argument('param_file', type=str, help='The model configuration file.')
-    parser.add_argument('description', type=str, help='A description for the experiment.')
+    parser.add_argument('--desc', type=str, required=True, help='A description for the experiment.')
+    parser.add_argument('--debug', action='store_true', help='Print verbose stack traces on error.')
+    parser.add_argument('--detach', action='store_true', help='Run experiment in the background and print its ID.')
+    parser.add_argument('--env', action='append', help='Set environment variables (e.g. NAME=value or NAME)')
+    parser.add_argument('--mount', action='append', help='Bind a host directory (e.g. /host/path:/target/path)')
+    parser.add_argument('--source', action='append', help='Bind a remote data source (e.g. source-id:/target/path)')
+    parser.add_argument('--cpu', help='CPUs to reserve for this experiment (e.g., 0.5)')
```

Comment:
If we're letting people specify this, we might as well also let them specify a GPU count, defaulting to 1.  This will let people not use a GPU if they don't need it, and will be future-proof for when people start using multiple GPUs.